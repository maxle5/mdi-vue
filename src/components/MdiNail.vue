
<template>
  <svg :width="size" :height="size" viewBox="0 0 24 24">
      <path d="M13.5,5V16L12,22L10.5,16V5H13.5M17,2H7V4H17V2Z" style="fill: currentColor" />
  </svg>
</template>

<script lang="ts">
  import { defineComponent } from 'vue';

  export default defineComponent({
      name: "MdiNail",
      props: {
          size: { 
              type: Number, 
              default: 24 
          }
      },
  })
</script>
