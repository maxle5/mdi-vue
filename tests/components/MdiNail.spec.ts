
import { mount } from "@vue/test-utils";
import { expect, test } from "vitest";
import MdiNail from "../../src/components/MdiNail.vue";

test("MdiNail snapshot", () => {
  const wrapper = mount(MdiNail, {});
  expect(wrapper.html()).toMatchSnapshot();
});
