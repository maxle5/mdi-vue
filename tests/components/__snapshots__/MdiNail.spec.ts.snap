// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MdiNail snapshot 1`] = `
"<svg width="24" height="24" viewBox="0 0 24 24">
  <path d="M13.5,5V16L12,22L10.5,16V5H13.5M17,2H7V4H17V2Z" style="fill: currentColor;"></path>
</svg>"
`;
